pep-0678.rst

```diff
@@ -189,8 +200,8 @@ or merely significant difficulties working out which explanation corresponds to
 which error. The new ``ExceptionGroup`` type intensifies these existing
 challenges.
 
-Keeping the ``__note__`` attached to the exception object, like the traceback,
-eliminates these problems.
+Keeping the ``__notes__`` attached to the exception object, like the
+traceback, eliminates these problems.
```

Comment:
```suggestion
Keeping the ``__notes__`` attached to the exception object, like the
traceback currently is, eliminates these problems.
```

This was slightly ambiguous to me on first read as to the intended meaning of "like the traceback"; this is intended to clarify that.